chain/chunks/src/lib.rs

```diff
@@ -1961,6 +1967,12 @@ impl ShardsManager {
     }
 
     pub fn handle_network_request(&mut self, request: ShardsManagerRequestFromNetwork) {
+        let _span = tracing::debug_span!(
+            target: "chunks",
+            "shards_manager_request_from_network",
+            "type" = <&ShardsManagerRequestFromNetwork as Into<&'static str>>::into(&request)
```

Comment:
nit: I think both of these work?

`Into::<&'static str>::into(&request)`
`<&'static str>::from(&request)`

Reply:
Ah yes, that's much nicer, thanks.